src/test/java/com/wikia/webdriver/testcases/mcfootertests/EnAnonMixedContentFooterTests.java

```diff
@@ -0,0 +1,91 @@
+package com.wikia.webdriver.testcases.mcfootertests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.mcfooter.MixedContentFooter;
+
+import org.testng.annotations.Test;
+
+@Test (groups = {"EnAnonMixedContentFooter"})
+@Execute(onWikia = "gameofthrones", asUser = User.ANONYMOUS)
+public class EnAnonMixedContentFooterTests extends NewTestTemplate{
+
+  @Test
+   public void mcFooterIsPresentOnENwiki() {
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isMcfooterPresent());
+  }
+
+  @Test
+   public void exploreWikisCardIsPresentOnENwiki(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isExploreWikisCardPresent());
+  }
+
+  @Test
+  public void discussionsCardIsPresentOnENwikiWithDiscussions(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
+    MCFooter.openWikiMainPage().scrollToFooter();
+    Assertion.assertTrue(MCFooter.isDiscussionsCardPresent());
+  }
+
+  @Test
+  @Execute(onWikia = "agas")
+  public void discussionsCardIsPresentOnENwikiWithEmptyDiscussions(){
+    MixedContentFooter MCFooter = new MixedContentFooter(driver);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00117)